packages/windbg.vm/tools/chocolateyinstall.ps1

```diff
@@ -5,16 +5,22 @@ try {
     $toolName = 'WinDbg'
     $category = 'Debuggers'
 
-    # It seems WinDbg is now distributed as an .appinstaller and we need to install it using Add-AppxPackage
-    Add-AppxPackage -AppInstallerFile 'https://windbg.download.prss.microsoft.com/dbazure/prod/1-0-0/windbg.appinstaller'
+    $exeUrl = "https://windbg.download.prss.microsoft.com/dbazure/prod/1-2402-24001-0/windbg.msixbundle"
+    $exeSha256 = "e941076cb4d7912d32a22ea87ad2693c01fa465227b4d1ead588283518de428f"
 
-    $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
-    $shortcut = Join-Path $shortcutDir "$toolName.lnk"
-    $executableCmd  = Join-Path ${Env:WinDir} "system32\cmd.exe"
-    # Use `start` to close the open console
-    $executableArgs = "/C start WinDbgX.exe"
-    $executableDir  = Join-Path ${Env:UserProfile} "Desktop"
-    Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executableCmd -Arguments $executableArgs -WorkingDirectory $executableDir -RunAsAdmin
+    $packageArgs = @{
+        packageName   = ${Env:ChocolateyPackageName}
+        url           = $exeUrl
+        checksum      = $exeSha256
+        checksumType  = "sha256"
+        fileFullPath  = Join-Path ${Env:TEMP} "$toolName.msixbundle"
+    }
+    Get-ChocolateyWebFile @packageArgs
+    Add-AppxPackage -Path $packageArgs.fileFullPath
```

Comment:
What about simplifying this code using the MSIX bundle URL directly in `Add-AppxPackage` instead of downloading it? As the URL includes the version, the change still allows versioning of Windbg, but with a simpler code.
```PowerShell
Add-AppxPackage "https://windbg.download.prss.microsoft.com/dbazure/prod/1-2402-24001-0/windbg.msixbundle"
```

Reply:
For me, this doesn't mean you shouldn't check the hash value. Connection failure could have an impact on receiving the full file or someone could carry out a MITM attack. I can remove this if you think it's necessary but for me, the code overhead isn't that much, in addition to being consistent with other packages.

Reply:
You are right that the extra check ensures we are installing what we expected and the code is not very complicated. So you have convinced me we should leave this code :wink: 

But I want to clarify that we are not checking the hash for all packages:
- For some packages that use a URL without the version, get updated often and install tools that are important from a "trusted" URL, we are using `VM-Assert-Signature` to ensure we can install the package without checking the hash (which breaks the package often). `VM-Assert-Signature` which checks that there is a valid signature and that the certificate was not signed by a publisher not trusted on the system. This is not ideal and @emtuls is proposing checking the signing authority in https://github.com/mandiant/VM-Packages/pull/1007, but we need to discuss how we want to handle this in more detail. Please feel free to share your thoughts and ideas on this!
- packages installed with `pip` and `npm`

Any ideas to improve the installation/verification of tools is very appreciated. :smile: